aibolit/__init__.py

```diff
@@ -1,9 +1,9 @@
 # SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
 # SPDX-License-Identifier: MIT
 
-"""
+'''
```

Comment:
In the original discussion of the issue, I failed to differentiate between the regular string and the docstring. PEP8 leaves it up to the user to select types of quotes for a regular string. We prefer single quotes here. But PEP257 https://peps.python.org/pep-0257/ clearly demands docstring triple quotes to be typed with single quote signs only.

@yegor256 please highlight your preferences on the docstings. We either follow pep257 or not.